pkg/frontend/clustermanager_type.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+)
+
+func (f *frontend) readOcmResourceType(vars map[string]string) error {
+	badRequestError := fmt.Errorf("the resource type '%s' could not be found in the namespace '%s' for api version '%s'", vars["ocmResourceType"], vars["resourceProviderNamespace"], vars["api-version"])
```

Comment:
Would it be better to remove it from the error or add a check for it?

Reply:
If it is something we need to validate then I would add the validation, I'm just not sure why we say it can't be found in the namespace when we have an explicit list of resourceTypes we support.